src/libraries/Microsoft.PowerFx.Core/Functions/UserDefinedFunction.cs

```diff
@@ -34,6 +34,26 @@ internal class UserDefinedFunction : TexlFunction
 
         public TexlNode UdfBody { get; }
 
+        public int GetArgIndex(string argName)
+        {
```

Comment:
organization nit: move after the constructor and probably after the main functionality (bind/checktypes)